src/main/java/frc/robot/AutoCommands.java

```diff
@@ -290,9 +290,13 @@ private Command autoAlignCoral(
               alignToReef::atDistanceGoal,
               alignToReef::atLateralGoal,
               alignToReef::atThetaGoal))
-        .withDeadline(
-            parallel(armSubsystem.run(armMethod).until(armSubsystem::isAtPosition), alignToReef)
-                .andThen(parallel(driveCommand, armSubsystem.run(armMethod))))
+        .withDeadline(parallel(armSubsystem.run(armMethod).until(armSubsystem::isAtPosition), alignToReef))
+        .andThen(
+            parallel(
+                driveCommand,
+                  armSubsystem.run(armMethod),
+                  ledSubsystem.runPatternAsCommand(LEDPattern.solid(Color.kGreen))))
+        .until(() -> !armSubsystem.hasCoral() || !driveCommand.isScheduled())
```

Comment:
I'm not sure where this `until` condition came from. It wasn't in the code on `main`, and I don't think we need it. It should keep running until the driver releases the button